### PR TITLE
Fix the issue #826

### PR DESCRIPTION
**Pull request by Anishka0107**
Fix for the issue https://github.com/openaustralia/planningalerts/issues/826.
Yahoo! Pipes does not exist anymore. So, a good alternative to it seems to be Zapier https://zapier.com/ 
The link for Yahoo Pipes on the Get the Data - API page is fixed.

### CODE REVIEW

**Comment by Anishka0107** [comment]
I am unable to understand why the checks are unsuccessful. Can somebody please help me here?

**Comment by equivalentideas** [comment]
> I am unable to understand why the checks are unsuccessful. Can somebody please help me here?

That check is the tests failing. But those failures have nothing to do with the change you made, so all good 👍 Those tests were added recently and maybe they're a bit brittle.

File: app/views/api/howto.html.haml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 %p
   Planning application data is available programmatically as feeds which can
-  be used in most web mapping APIs and desktop GIS software like #{link_to "Yahoo Pipes", "http://pipes.yahoo.com/"}. Details of the API are listed below.
+  be used in most web mapping APIs and desktop GIS software like <a href="https://zapier.com/">Zapier</a>. Details of the API are listed below.
```
Comment (equivalentideas) [inline app/views/api/howto.html.haml]: This should use the [Rails link_to helper](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) like the original.

File: app/views/applications/show.html.haml [hunk 2]
```diff
@@ -14,7 +14,7 @@
   #application
     %article#application-info
       %header
-        %h3.address= @application.address
+        %h1.address= @application.address
```
Comment (equivalentideas) [inline app/views/applications/show.html.haml]: This change is related to a different issue that #826 so it shouldn't be part of this PR (that would be an awesome issue to fix in a separate PR though 🎉 ).

File: app/views/api/howto.html.haml (hunk 1 above)
Comment (Anishka0107) [inline app/views/api/howto.html.haml]: As you said sir,
Updated <a> to link_to and also changed Zapier to Superpipes as it seems a better alternative and is open-source as well.

File: app/views/applications/show.html.haml (hunk 2 above)
Comment (Anishka0107) [inline app/views/applications/show.html.haml]: I have changed it back to the original one and will submit a separate PR when this PR gets merged.
